charts/prometheus-mysql-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for prometheus mysql exporter with cloudsqlproxy
 name: prometheus-mysql-exporter
-version: 1.14.0
+version: 2.0.0
 home: https://github.com/prometheus/mysqld_exporter
 appVersion: v0.14.0
```

exporter could be updated too

```suggestion
appVersion: v0.15.0
```